lib/level10/games/player.ex

```diff
@@ -1,9 +1,9 @@
 defmodule Level10.Games.Player do
-  defstruct [:key, :name]
+  defstruct [:id, :name]
 
   def new(name) do
     %__MODULE__{
-      key: Ecto.UUID.generate(),
+      id: Ecto.UUID.generate(),
```

Comment:
I envisioned the key being a key that we would give the player's client so that they could re-enter the game if they left or something. More of an auth token. If we're adding an ID to the players, would it make sense to keep the key field in place, or rename it to something more obvious?

Reply:
Oh I wasn't thinking of it like a token

I thought you just meant an ID for reconnecting
What do you want me to do?